server/lib/games.ts

```diff
@@ -245,14 +239,29 @@ export function getCurrentRoundscores(lobbyNr: number): number[] {
   });
 }
 
-export function getFirstActivePlayer(lobbyNr: number): ActivePlayer {
-  const currentGame = getGameByLobby(lobbyNr);
+export async function getFirstActivePlayer(
+  lobbyNr: number,
+  socketid: string
```

Comment:
Naming: How about socketID or socketId?

Reply:
Unfortunately in this case socketID is already taken, thats why I had to name it like this :(